tests/unit/io/eventlet_utils.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2013-2015 DataStax, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import os
+import select
+import socket
+import thread
+import Queue
+import threading
+import __builtin__
+import ssl
+import time
+
+
+def eventlet_un_patch_all():
```

Comment:
So I was not able to get the unpatching for eventlet to work in all cases. I will look at adding this to our test runs in tox.